cluster/src/test/scala/io/snappydata/benchmark/snappy/tpcds/TableCreationJob.scala

```diff
@@ -31,81 +47,104 @@ object TableCreationJob extends SnappySQLJob{
+    createColumnPartitionedTable(snSession, props, tableName)
 
-
-    props = Map(("PARTITION_BY" -> "inv_item_sk"), ("BUCKETS" -> buckets_ColumnTable))
+    partitionBy = "inv_item_sk"
+    props = Map(("PARTITION_BY" -> partitionBy), ("BUCKETS" -> buckets_ColumnTable))
     tableName = "inventory"
-    createColumnPartitionedTables(snSession, props, tableName)
+    createColumnPartitionedTable(snSession, props, tableName)
 
-    props = Map(("PARTITION_BY" -> "sr_item_sk"), ("BUCKETS" -> buckets_ColumnTable),
+    partitionBy = "sr_item_sk"
+    props = Map(("PARTITION_BY" -> partitionBy), ("BUCKETS" -> buckets_ColumnTable),
       ("COLOCATE_WITH" -> "INVENTORY"))
     tableName = "store_returns"
-    createColumnPartitionedTables(snSession, props, tableName)
+    createColumnPartitionedTable(snSession, props, tableName)
 
-    props = Map(("PARTITION_BY" -> "ss_item_sk"), ("BUCKETS" -> buckets_ColumnTable),
+    partitionBy = "ss_item_sk"
+    props = Map(("PARTITION_BY" -> partitionBy), ("BUCKETS" -> buckets_ColumnTable),
       ("COLOCATE_WITH" -> "STORE_RETURNS"))
     tableName = "store_sales"
-    createColumnPartitionedTables(snSession, props, tableName)
-
+    createColumnPartitionedTable(snSession, props, tableName)
 
-    props = Map(("PARTITION_BY" -> "c_customer_sk"), ("BUCKETS" -> buckets_ColumnTable))
+    partitionBy = "c_customer_sk"
+    props = Map(("PARTITION_BY" -> partitionBy), ("BUCKETS" -> buckets_ColumnTable))
     tableName = "customer"
-    createColumnPartitionedTables(snSession, props, tableName)
+    createColumnPartitionedTable(snSession, props, tableName)
 
-    props = Map(("PARTITION_BY" -> "ca_address_sk"), ("BUCKETS" -> buckets_ColumnTable))
+    partitionBy = "ca_address_sk"
+    props = Map(("PARTITION_BY" -> partitionBy), ("BUCKETS" -> buckets_ColumnTable))
     tableName = "customer_address"
-    createColumnPartitionedTables(snSession, props, tableName)
+    createColumnPartitionedTable(snSession, props, tableName)
 
-    val avgFileStream: FileOutputStream = new FileOutputStream(
-      new File(s"Snappy_Average.out"))
-    val avgPrintStream: PrintStream = new PrintStream(avgFileStream)
+    // cleanup
+    loadPerfPrintStream.flush()
+    loadPerfFileStream.flush()
+    loadPerfFileStream.close()
+    loadPerfFileStream.close()
 
   }
 
-  def createColumnPartitionedTables(snappy: SnappySession,
-      props: Map[String,String] , tableName: String): Unit = {
+  def createColumnPartitionedTable(snappy: SnappySession,
+                                   props: Map[String, String], tableName: String): Unit = {
 
     val df = snappy.read.parquet(s"$dataLocation/$tableName")
+    snappy.dropTable(tableName, ifExists = true)
     snappy.createTable(tableName, "column",
       new StructType(df.schema.map(_.copy(nullable = false)).toArray), props)
     df.write.insertInto(tableName)
-    val cnt = df.collect().length
+    // val cnt = df.collect().length
```

Comment:
Please delete line if not required.

Reply:
The job ran very slow due to that old line of code. Kept it around to indicate that. Will add a comment.